docs/user/index.rst

```diff
@@ -0,0 +1,120 @@
+######################
+Using Aldryn Reversion
+######################
+
+After you set up and configured your models (they are registered for revisions
+support) you can start using ``aldryn-reversion`` end-user features.
+
+************************
+Restoring object version
+************************
+
+Most likely you already know that django keeps track of object changes also
```

Comment:
How about:
"Django keeps track of changes to objects and makes them available, as history, for the object's admin edit form. With ``aldryn-reversion`` that history has a few more features. As soon as the object is created, an 'initial version' is created and treated as the first restore point."
?